storm-core/src/storm.thrift

```diff
@@ -222,6 +222,86 @@ struct TopologyInfo {
 514: optional string owner;
 }
 
+struct CommonAggregateStats {
+513: optional i32 num_executors;
```

Comment:
Can we perhaps make all of these start with a 1?  There were other places that I used a 512 for security to avoid a collision with others possibly making changes at the same time while it was on a different branch, but I don't think we have that problem here.